syntax = "proto3";

message JoinInfo {
  bytes nickname = 1;
  bytes mail = 2;
}

message ChatInfo {
  bytes sender = 1;
  bytes content = 2;
}

message RoomInfo {
  bytes creator = 1;
  bytes creator_mail = 2;
  bytes room_mail = 3;
}
